extensions/Worker.Extensions.Storage.Tables/src/Worker.Extensions.Storage.Tables.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <AssemblyName>Microsoft.Azure.Functions.Worker.Extensions.Storage.Tables</AssemblyName>
+    <RootNamespace>Microsoft.Azure.Functions.Worker.Extensions.Storage.Tables</RootNamespace>
+    <Description>Azure Table Storage extensions for .NET isolated functions</Description>
+    
+    <!--Version information-->
+    <VersionPrefix>1.0.0-preview1</VersionPrefix>
```

Comment:
nit: Can use the `VersionSuffix` element.
`<VersionSuffix>-preview1</VersionSuffix>`